examples/kubernetes/connectivity-check/Makefile

```diff
@@ -5,6 +5,8 @@ include ../../../Makefile.defs
 include ../../../Makefile.quiet
 
 DEFAULT_OUT := connectivity-check.yaml
+# Same as connectivity-check.yaml, but without 1.1.1.1 and google.com
+IPv6_ONLY_OUT := connectivity-check-ipv6.yaml
```

Comment:
nit: I wonder if it's worth calling this `internal` or `cluster` or similar given that the actual implementation is not changing to guarantee ipv4 / ipv6, it's just turning on/off external checks.

Reply:
This is really better naming, let me make changes for this.

Reply:
I have made changes, please help to take another :eyes: whenever you have time. Thanks :)